tests/BenchmarkDotNet.IntegrationTests/MonoTests.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Environments;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Tests.XUnit;
+
+namespace BenchmarkDotNet.IntegrationTests
+{
+    public class MonoTests : BenchmarkTestExecutor
+    {
+        [FactDotNetCoreOnly("UseMonoRuntime option is available in .NET Core only starting from .NET 6")]
+        public void Mono60IsSupported()
+        {
+            var config = ManualConfig.CreateEmpty().AddJob(Job.Dry.WithRuntime(MonoRuntime.Mono60));
+            CanExecute<MonoBenchmark>(config);
+        }
```

Comment:
Not sure if there is a need to add something like
```csharp
if (ContinuousIntegration.IsAppVeyorOnWindows()) // too time consuming for AppVeyor (1h limit)
                return;
```
https://github.com/dotnet/BenchmarkDotNet/blob/master/tests/BenchmarkDotNet.IntegrationTests/NativeAotTests.cs#L24

Reply:
If AppVeyor CI builds are passing, we don't need it, we can always add it in the future when it becomes a problem,